README.md

```diff
@@ -1,4 +1,32 @@
-# Github Templates
+# .github
 
-This repo hosts a few templates to be applied across all repos in this organisation.
-It also holds the organisation's profile, which is public to everyone.
+This repository contains organization-wide GitHub configurations and community health files for the Overture project. These files serve as default community health files for all Overture repositories that don't have their own specific versions.
```

Comment:
I've noticed a few "code of conduct" files in other repos. Do those need "specific versions" of that? I think they can be removed from there.

Reply:
All code of conducts have been appropriately updated

Reply:
agreed and updating now